src/locale/zh_Hant/LC_MESSAGES/django.po

```diff
@@ -2427,7 +2427,7 @@ msgid ""
 "focuses on Python technology and its versatile applications. We welcome "
 "people who are interested in Python to join PyCon Taiwan to share knowledge,"
 " exchange ideas, make connections and to help us grow our network.</p>"
-msgstr "<p>PyCon Taiwan 為一年一度由愛好者舉辦、討論並提倡使用 Python 程式語言的會議，聚焦在 Python 技術與其多樣的可能應用的交流。我們歡迎所有對 Python 有興趣的朋友一同加入 PyCon Taiwan 來分享所學、交換想法、並且認識更多同好。 </p>"
+msgstr "<p>PyCon Taiwan 為一年一度由愛好者舉辦、討論並提倡使用 Python 程式語言的會議，聚焦在 Python 技術與其多樣的可能應用的交流。我們歡迎所有對 Python 有興趣的朋友一同加入 PyCon Taiwan 來分享所學、交換想法、並且認識更多同好。 </p>"
```

Comment:
Please don't edit the `.po` file by your own.

We use `transifex` in this project so all the translation will be done after merge.